source/locale/ja/LC_MESSAGES/concepts/reward-programs.po

```diff
@@ -42,7 +42,7 @@ msgid ""
 " Reward Programs** aimed at further encouraging users to create and maintain"
 " nodes."
 msgstr ""
-"ノードがブロックを生成すると取得する **ブロック報酬** (:doc:`手数料` と :doc:`インフレーション`) による直接な報酬の他に、 "
+"ノードがブロックを生成すると取得する **ブロック報酬** ( :doc:`手数料` と :doc:`インフレーション` ) による直接な報酬の他に、 "
```

Comment:
```suggestion
"ノードがブロックを生成すると取得する **ブロック報酬** ( :doc:`fees` と :doc:`inflation` ) による直接な報酬の他に、 "
```

This is a filename, you cannot translate it:
:doc:`something` -> DO NOT TRANSLATE
:doc:`something1 <something2>` -> TRANSLATE something1 BUT NOT something2

Same thing for :ref:.

Reply:
Thanks. I will fix them on Transifix and push commit again.